security/pfSense-pkg-sudo/files/usr/local/pkg/sudo.inc

```diff
@@ -203,4 +206,73 @@ function sudo_validate_commands(&$input_errors) {
 		}
 	}
 }
+
+function sudo_sync_config() {
+	global $config;
+	if (is_array($config['hasync'])) {
+		$system_carp = $config['hasync'];
+		// XMLRPC sync is currently only supported over connections using the same protocol and port as this system
+		if ($config['system']['webgui']['protocol'] == "http") {
+			$varsyncprotocol = "http";
+			$varsyncport = $config['system']['webgui']['port'] ?: '80';
+		} else {
+			$varsyncprotocol = "https";
+			$varsyncport = $config['system']['webgui']['port'] ?: '443';
+		}
+		sudo_do_xmlrpc_sync($system_carp['synchronizetoip'], $varsyncport, $varsyncprotocol, $system_carp['username'], $system_carp['password'], 100);
```

Comment:
Please break this really long line.  pfSense style says we should try to keep lines in ~80 columns.  Multiline statements must be indented with 4 spaces after previous line indentation.